### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by boekhorstb1**
- updateing README.md
- updateing docs/ files for osism and scs websites

Depends: #73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,23 +29,77 @@ git clone https://github.com/SovereignCloudStack/rookify
...
+5. To install the container-based environment
 ```bash
 make build-container
 docker run -ti --mount type=bind,source="$(pwd)",target=/app/rookify/src/,readonly --workdir=/app/rookify/src rookify:latest
 ```
 
 ## Usage
-__TODO__
+
+**NOTE**: for testing purposes the [OSISM Testbed](https://github.com/osism/testbed) is used. The `Makefile` and example configuratino (`config.example.osism.yaml`) have built in helper functions and examples for this test-setup.
```
Comment (NotTheEvilOne) [inline README.md:51]: Typo

File: README.md [hunk 2]
```diff
@@ -29,23 +29,77 @@ git clone https://github.com/SovereignCloudStack/rookify
...
+
+```
+ls config.example.*
+# there is a specific example config for the osism testbed: config.example.osism.yaml
+cp config.example.yml config.yaml
+```
+
+_Adapt the config.yml to your need_: especially enter the correct paths for ssh-keys, kubernetes configuration and ceph configuration (all these configuration files need to be provided!).
+Note:
+    - for the testbed there is a helper script to download the configs from the testbed. These helperscripts need correct `.ssh/config` entries to work (take a look at [scripts/get_configs_from_testbed](scripts/get_configs_from_testbed.sh) for an example).
```
Comment (NotTheEvilOne) [inline README.md:65]: Please move the provided testbed scripts to a `osism` sub-directory. :)

File: README.md [hunk 3]
```diff
@@ -29,23 +29,77 @@ git clone https://github.com/SovereignCloudStack/rookify
...
+    - the helper scripts are merely there to help for testing with the [OSISM testbed](https://github.com/osism/testbed) and might not suit your purposes.
+
+### Provide needed configuration files from target servers
```
Comment (NotTheEvilOne) [inline README.md:68]: This section should be moved one up as this is the generic variant of the scripted one described.

File: README.md [hunk 4]
```diff
@@ -29,23 +29,77 @@ git clone https://github.com/SovereignCloudStack/rookify
...
+run-rookify
+```
+
+### Check the other options
 
 Type `make` to get a list of available development specific commands.
 
+## Troubleshooting
+
+### OSISM Testbed
```
Comment (NotTheEvilOne) [inline README.md:94]: Please move this section below the generic one.

File: README.md [hunk 5]
```diff
@@ -29,23 +29,77 @@ git clone https://github.com/SovereignCloudStack/rookify
...
+
+**ssh-issues:**
+    - make sure the id-rsa keys are "clean" and do not contain unexpected strings like "\<\<EOF"
```
Comment (NotTheEvilOne) [inline README.md:97]: An alternative solution would be to call `ssh-keygen -p -N "" -f ssh.key` to convert and reformat the keyfile to the expected format.

**Review by NotTheEvilOne — CHANGES_REQUESTED** [review]
Please create a separate PR for the `Makefile` changes.

**Comment by boekhorstb1** [comment]
> Please create a separate PR for the `Makefile` changes.

ok I added makefile updates into this pr:

https://github.com/SovereignCloudStack/rookify/pull/73

File: README.md
Comment (NotTheEvilOne) [inline README.md:1]: Please replace the following line:
`Kubernetes nodes should be rolled out at least on the OSD nodes`
with:
`Kubernetes nodes should be rolled out at least on the monitor nodes`

File: README.md [hunk 6]
```diff
@@ -29,25 +30,90 @@ git clone https://github.com/SovereignCloudStack/rookify
...
+
+### Provide needed configuration files from target servers
+
+Copy the needed configuration-files from the servers that need to be migrated from ceph to rook.
+
+_Provide needed configuration files as written in the config.yml._ At least required are:
+- ./ceph/ceph.conf (typically found in `/etc/ceph/` on a testbednode)
+- ./ceph/ceph.admin.keyring (typically found in `/etc/ceph/` on a testbednode)
+- kubernetes config of user (e.g. found in `~/.kube/config`)
+- ssh key of the server (for the testbed they are typically found in `./terraform/.id.rsa` folder of the testbed-repository)
```
Comment (NotTheEvilOne) [inline README.md:73]: "found at `./terraform/.id.rsa` in the `testbed` directory"

**Review by NotTheEvilOne — CHANGES_REQUESTED** [review]
Other than that the changes look good so far.

**Review by b1-lender — APPROVED** [review]

File: README.md
Comment (boekhorstb1) [reply README.md:1]: thanks! changed that

**Review by NotTheEvilOne — APPROVED** [review]
LGTM

**Pull request merged**